src/builder.py

```diff
@@ -275,19 +389,86 @@ def run(config: state.BuilderRunConfig, proxy: state.ProxyConfig | None) -> list
         raise BuilderRunError from exc
 
 
-def get_latest_image(arch: state.Arch, base: state.BaseImage, cloud_name: str) -> str:
+def _get_image_name(arch: state.Arch, base: state.BaseImage, prefix: str) -> str:
+    """Get image name.
+
+    Args:
+        arch: The image compute architecture.
+        base: The Ubuntu OS base of the image.
+        prefix: The prefix to use for image name.
+
+    Returns:
+        The image name to upload on OpenStack.
+    """
+    return f"{prefix}-{base.value}-{arch.value}"
```

Comment:
Is a method really necessary for a single line f-string ?

Reply:
What bothers me is that it's adding 12 lines for a method that only concatenates 3 strings with a "-" used 2 times in the code.
Also the image_name seems to be quite linked to the "RunConfig", can't we add an attribute or a property to the "RunConfig" dataclass that return the image name ?

Reply:
That's a good point, i'll move it as an attribute of the RunConfig and move the prefix config over to image config.